### PR TITLE
[CLOUDTRUST-1917] whitelist 2 more messages errors

### PR DESCRIPTION
**Pull request by bsoniam**

### CODE REVIEW

File: keycloak_client.go [hunk 1]
```diff
@@ -413,7 +413,7 @@ func whitelistErrors(statusCode int, message string) error {
 			Message: "keycloak." + message,
 		}
 	// update account in back-office or self-service
-	case (message == "User exists with same username or email") || (message == "usernameExistsMessage") || (message == "emailExistsMessage"):
+	case (message == "User exists with same username or email") || (message == "usernameExistsMessage") || (message == "emailExistsMessage") || (message == "Username exists with same username") || (message == "Username exists with same email"):
```
Comment (fperot74) [inline keycloak_client.go]: This line starts to be a little bit too long...
You should use a static map with white-listed messages as keys

Comment (Fratt) [reply keycloak_client.go]: But... Will I receive an "error code" as well, something like keycloak-bridge.username-exists ?
Or will I receive a full sentence in english ?

Comment (bsoniam) [reply keycloak_client.go]: you will not receive a full sentence, nothing changed in how to format our message errors

Comment (bsoniam) [reply keycloak_client.go]: @fperot74 , ok now?

Comment (Fratt) [reply keycloak_client.go]: great :-)
Can you give me the list of all "new error codes" I could receive?
thanks!

Comment (bsoniam) [reply keycloak_client.go]: it should be keycloak.existing.UsernameOrEmail

**Pull request merged**